cmd/csi-attacher/main.go

```diff
@@ -76,6 +76,8 @@ var (
 
 	kubeAPIQPS   = flag.Float64("kube-api-qps", 5, "QPS to use while communicating with the kubernetes apiserver. Defaults to 5.0.")
 	kubeAPIBurst = flag.Int("kube-api-burst", 10, "Burst to use while communicating with the kubernetes apiserver. Defaults to 10.")
+
+	grpcLogCharCap = flag.Int("grpc-log-char-cap", -1, "The maximum amount of characters logged for every grpc responses. Defaults to no limit")
```

Comment:
Maybe name this `max-grpc-log-length`? This would match the grpc function more closely.

Reply:
done